dev/source/docs/ros-distance-sensors.rst

```diff
@@ -0,0 +1,72 @@
+.. _ros-distance-sensors:
+
+==========================
+ROS distance sensors usage
+==========================
+
+.. warning::
+
+    Distance sensors message are currently support on :
+
+    - Ardupilot Master
+    - Copter >= 3.5.1
```

Comment:
nope, on copter, the input is accepted from 3.4. Even on 3.5, copter won't send the msg for rangefinder. It is only send with proximity that is another usage (Didn't write it yet)